src/legacy/ui/public/saved_objects/components/saved_object_save_modal.tsx

```diff
@@ -16,249 +16,10 @@
-      isLoading: true,
-    });
-
-    await this.props.onSave({
-      newTitle: this.state.title,
-      newCopyOnSave: this.state.copyOnSave,
-      isTitleDuplicateConfirmed: this.state.isTitleDuplicateConfirmed,
-      onTitleDuplicate: this.onTitleDuplicate,
-    });
-  };
-
-  private onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    this.setState({
-      title: event.target.value,
-      isTitleDuplicateConfirmed: false,
-      hasTitleDuplicate: false,
-    });
-  };
-
-  private onCopyOnSaveChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    this.setState({
-      copyOnSave: event.target.checked,
-    });
-  };
-
-  private onFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault();
-    this.saveSavedObject();
-  };
-
-  private renderDuplicateTitleCallout = () => {
-    if (!this.state.hasTitleDuplicate) {
-      return;
-    }
-
-    return (
-      <Fragment>
-        <EuiCallOut
-          title={
-            <FormattedMessage
-              id="common.ui.savedObjects.saveModal.duplicateTitleLabel"
-              defaultMessage="A {objectType} with the title '{title}' already exists."
-              values={{ objectType: this.props.objectType, title: this.state.title }}
-            />
-          }
-          color="warning"
-          data-test-subj="titleDupicateWarnMsg"
-        >
-          <p>
-            <FormattedMessage
-              id="common.ui.savedObjects.saveModal.duplicateTitleDescription"
-              defaultMessage="Click {confirmSaveLabel} to save the {objectType} with the duplicate title."
-              values={{
-                objectType: this.props.objectType,
-                confirmSaveLabel: (
-                  <strong>
-                    <FormattedMessage
-                      id="common.ui.savedObjects.saveModal.duplicateTitleDescription.confirmSaveText"
-                      defaultMessage="Confirm Save"
-                    />
-                  </strong>
-                ),
-              }}
-            />
-          </p>
-        </EuiCallOut>
-        <EuiSpacer />
-      </Fragment>
-    );
-  };
-
-  private renderCopyOnSave = () => {
-    if (!this.props.showCopyOnSave) {
-      return;
-    }
-
-    return (
-      <Fragment>
-        <EuiSwitch
-          data-test-subj="saveAsNewCheckbox"
-          checked={this.state.copyOnSave}
-          onChange={this.onCopyOnSaveChange}
-          label={
-            <FormattedMessage
-              id="common.ui.savedObjects.saveModal.saveAsNewLabel"
-              defaultMessage="Save as a new {objectType}"
-              values={{ objectType: this.props.objectType }}
-            />
-          }
-        />
-        <EuiSpacer />
-      </Fragment>
-    );
-  };
-}
+/**
+ * @deprecated
+ *
+ * Do not import this component from here. Improt from `src/plugins/kibana_react` instead.
+ */
+export { SavedObjectSaveModal } from '../../../../../plugins/kibana_react/public';
```

Comment:
Since this is 100% the same as the existing component, isn't it worth just updating the import paths everywhere this component is used?

Reply:
I will refactor Embeddables (and possibly Visualize) in a subsequent PR to use the new import, but it will not be just updating paths.